frontend/libs/studio-components/src/components/StudioRecommendedNextAction/StudioRecommendedNextAction.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import type { Meta, StoryFn } from '@storybook/react';
+import { fn } from '@storybook/test';
+import { StudioRecommendedNextAction } from './index';
+import type { StudioRecommendedNextActionProps } from './index';
+import { StudioIconTextfield } from '../StudioIconTextfield';
+import { KeyVerticalIcon } from '../../../../studio-icons/src';
+
+type PreviewProps = Omit<StudioRecommendedNextActionProps, 'children'>;
+
+type PreviewStory = StoryFn<PreviewProps>;
+
+const ComposedPreviewComponent = (props: PreviewProps) => {
+  return (
+    <StudioRecommendedNextAction {...props}>
+      <p>StudioRecommendedNextAction children will appear here</p>
+    </StudioRecommendedNextAction>
+  );
+};
+
+const meta: Meta = {
+  title: 'Studio/StudioRecommendedNextAction',
+  component: ComposedPreviewComponent,
+  args: {
+    onSave: fn(),
+    onSkip: fn(),
+  },
+};
+export const Preview: PreviewStory = (args): React.ReactElement => (
+  <ComposedPreviewComponent {...args} />
+);
+
+Preview.args = {
+  title: 'Recommended action title',
+  description: 'Recommended action description',
+  saveButtonText: 'Save',
+  skipButtonText: 'Skip',
+  hideSaveButton: false,
+};
+
+type ExampleUseCase = StoryFn<typeof StudioRecommendedNextAction>;
+export const ExampleUseCase: ExampleUseCase = (): React.ReactElement => {
+  const [name, setName] = useState('');
+  const onSave = () => {};
+
+  const onSkip = () => {};
```

Comment:
You can use the fn function from Storybook to capture events, which will then be displayed in the Action tab when the onClick event is triggered.
```suggestion
   import { fn } from '@storybook/test';
   
   const [name, setName] = useState('');
  const onSave = fn() ;
  const onSkip = fn();
```

Reply:
I see i've implemented it only for the main preview story and not the example case, good catch 👍